test/check.js

```diff
@@ -7,5 +7,5 @@ var postcss = require('postcss');
 module.exports = function check( input, output, opts ) {
   var processor = postcss([lost(opts)]);
 
-  expect(processor.process(input).css).to.equal(output);
```

Comment:
So looking at it again I still think there is something going on here that's not correct. I don't think it's the syntax that you changed...but I'm not sure what it is.

Here's a screencast that explains my thoughts.
http://quick.as/2LkHp0rQ